### ISSUE TITLE
Missing genitalia type?

### ISSUE THREAD

**Issue by Wortzenhein** [start-5/TiTS.JS-Save-Editor#81]
I know it's probably stupid to bring up? Or maybe paranoia, but I noticed the mothrine penis type is missing in the editor, wondering if that could be added?

**Comment by start-5** [comment]
You're right, that type is missing, and from what I'm seeing probably others. There are some checks missing for types that are a little harder to catch.

**Comment by Wortzenhein** [comment]
Ya, that was the one I was aware of, though I did think others were but just couldn't remember what ones. Just kinda surprised that one was missed cause it's on the main wiki for trials haha. Sorry if that sounds rude... didn't mean that

**Comment by kanjiirahlni** [comment]
I just made a thread about this issue too, didn't realize you made one.
